### ISSUE TITLE
The input box is enabled, and you can type in it

### ISSUE THREAD

**Issue by mercmobily** [vaadin/vaadin-date-picker#225]
This is seen in both Chrome and Firefox (both latest).

Create a standard element:

    <vaadin-date-picker></vaadin-date-picker>

You will see that the following can be done:

![1](https://cloud.githubusercontent.com/assets/2128734/18033864/9eb9e26e-6d60-11e6-809b-1bd09e009887.png)
![2](https://cloud.githubusercontent.com/assets/2128734/18033867/9ec0e56e-6d60-11e6-92f7-af01ccc7de3e.png)
![3](https://cloud.githubusercontent.com/assets/2128734/18033866/9ebfefce-6d60-11e6-9a2b-a16fd3ed6179.png)
![4](https://cloud.githubusercontent.com/assets/2128734/18033865/9ebf5dfc-6d60-11e6-9c91-9568b00cc286.png)

Basically, I use the date picker ti pick a date. All good. Then in screenshot 3 you can see that I moved the cursor using the arrows. And then I typed CHANGED there.

There is another level to this bug. If you delete what's in the textbox, and then click on the date picker again and select the SAME date again, you will see that the textbox does NOT get restored -- since technically speaking the date hasn't changed.

This is a showstopper for us, since the app will be used from very computer illiterate people...! Happy to help with a PR for a fix -- please direct me a little in the code though...

**Comment by mercmobily** [comment]
Actually, I am looking at the code, and I just cannot explain... why is there an input box at all...?

**Comment by mercmobily** [comment]
Well... I might not need a pointer in the code after all.
It looks like just changing:

    readonly$="[[readonly]]"

Into:

    readonly

in `vaadin-date-picker.html` totally fixes the issue. And it makes a lot of sense too.
If it makes sense to you too, I will do a PR, But, it would me a monumental waste of noise, network, database entries in Github, and log entries in your git repo... you might as well just press backspace 16 times on your editor!

**Comment by mercmobily** [comment]
Glad I could be of any help!

**Comment by mercmobily** [comment]
Did you release the fix? It's kind of crucial.. if so, feel free to close this!

**Comment by tomivirkki** [comment]
Hi @mercmobily,
There's a PR pending to be merged here https://github.com/vaadin/vaadin-date-picker/pull/226 Already gave it thumbs up, but we still need to fix an issue with the test environment to get the merge finished.

**Comment by mercmobily** [comment]
OK -- thanks for letting me know!
